packages/openssl/buildinfo.json

```diff
@@ -1,7 +1,7 @@
 {
   "single_source" : {
     "kind": "url_extract",
-    "url": "https://openssl.org/source/openssl-1.0.2p.tar.gz",
-    "sha1": "f34b5322e92415755c7d58bf5d0d5cf37666382c"
+    "url": "https://www.openssl.org/source/openssl-1.0.2r.tar.gz",
+    "sha1": "b9aec1fa5cedcfa433aed37c8fe06b0ab0ce748d"
```

Comment:
Confirmed that this SHA matches the one in https://www.openssl.org/source/openssl-1.0.2r.tar.gz.sha1